### ISSUE TITLE
maxLines default value

### ISSUE THREAD

**Issue by jasbindar-singh** [mikelpmc/react-multiline-clamping#32]
Problem: I have a use case where I want to show the whole string when the user clicks on `Show More` cta but maxLines prop default value caps it to 8 lines.
Temporary solution: I have a huge value on maxLines prop for now to fix that but it somehow seems very hacky
Question: Is there any other way I could achieve that like without having to specify a huge number for maxLines prop value

**Comment by lagroms** [comment]
Agreed have the same issue. This prop should be optional. @mikelpmc 

I've submitted a PR for this fix
